releasenotes/notes/conntrack-fail-fast-d721b42a2fea96ff.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Network monitor now fails to load if conntrack initialization fails on
+    system-probe startup. Set system_probe_config.ignore_conntrack_init_failure
```

Comment:
This should be scoped under network_config since this only affects the network module